projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/variants.a11y-e2e.cy.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { configureProductWithVariants } from '../../helpers/variants/apparel-checkout-flow';
+
+describe('Variants Accessibility test', { testIsolation: false }, () => {
+  before(() => {
+    cy.window().then((win) => {
+      win.sessionStorage.clear();
+    });
+    configureProductWithVariants();
+    cy.a11yContinuumSetup();
+    cy.requireLoggedIn();
+    cy.visit('/');
+  });
+
+  it('Product Details', () => {
+    cy.get('.Section4 cx-banner cx-generic-link a')
+      .first()
+      .scrollIntoView()
+      .click();
+    cy.get(
+      'cx-product-variants-container .variant-section'
+    ).a11yRunContinuumTest();
+  });
+});
```

Comment:
```suggestion
describe('Variants Accessibility test', { testIsolation: false }, () => {
  before(() => {
    configureProductWithVariants();
    cy.a11yContinuumSetup();
    cy.visit('/p/300785814');
  });

  it('Product Details', () => {
    cy.get(
      'cx-product-variants-container .variant-section'
    ).a11yRunContinuumTest();
  });
});
```

Do we need to be logged in for this test to work? I think we could simplify the setup a bit.

Reply:
You are 100% right. Thank you 👍